README.md

```diff
@@ -14,7 +14,21 @@ Additional description about the project and its features.
 
 ## Live Demo
 
-- [Live Demo Link](https://mstanley-me.github.io/toDoList/)
+- [Live Demo Link](https://mstanley-me.github.io/TODO/)
+
+
+## Getting Started
+
+- To get a local copy up and running follow these simple example steps.
+
+# Install
+
```

Comment:
- [OPTIONAL] As requested by the previous code reviewer, it will be nice add `steps(commands)` on how to clone your app to the local machine, before running it. Please kindly update your `readme file` with this `steps(commands/instructions)` as this will be of help to non technical users and some are potential recruiters.

Reply:
Okay thank you so much!😁
I apreciate !!!